### ISSUE TITLE
"Pocket wallet" powered by a smart contract

### ISSUE THREAD

**Issue by alescode** [gnosis/fund-recovery#4]
In today's banking-centric world, we can't easily access a huge amount of money instantly. People are used to this, and it makes sense for most, as it mitigates the risk of loss or theft.

We could make a "pocket wallet" app that controls the funds in a smart contract and automatically rejects payments that would tip its balance over a certain amount. If this type of wallet becomes commonplace, we could narrow our focus on fund recovery for wallets with more built-in restrictions: [Coinbase Vault](https://support.coinbase.com/customer/en/portal/articles/1612208-the-coinbase-vault-) was a big step in this direction. Biometrics, multisig, and third-party ID verification all start making more sense once we only try to recover vaults that are economically significant for the user, rather than pocket wallets that can only hold $20.

**Comment by tschubotz** [comment]
I agree, small amounts would not be worth recovering. For small amounts, the overhead of gathering multiple signatures is probably also unnecessary.
However, with the current volatile crypto prices, it's hard to argue with a user, what "small amount" means, since it could be worth much more, soon.

Our plans is to resolve this via a daily limit (specified in ETH or in fiat) which is the amount that can be spent by a single owner without requiring signatures of other owners.
The Gnosis MultiSig (https://wallet.gnosis.pm/) already has this feature and we are planning on keeping a similar version of it in the new Gnosis Safe smart contracts.

A timelock feature option for big transfers, similar to Coinbase Vault would be great too, I agree!
